src/components/canvas/labels/Labels.tsx

```diff
@@ -36,7 +36,7 @@ const Label = ({
 );
 
 const Labels = (): JSX.Element => {
-  const graph = useGraph();
+  const { graph } = useGraph();
```

Comment:
what purpose does destructureing this serve?

Reply:
The `useGraph` hook returns a javascript object containing a bunch of different fields, such as the `graph` itself and functions for updating the graph.

Components can then call the hook and extract only the fields they want from the return value.

That's the purpose of object destructuring here.

In this case, we are only interested in the `graph` field inside the return value of `useGraph()`, so we destructure the return value to extract only the graph field.